pkg/crc/preflight/preflight_checks_nondarwin.go

```diff
@@ -0,0 +1,25 @@
+// +build !darwin
+
+package preflight
+
+import (
+	"errors"
+
+	"github.com/code-ready/gvisor-tap-vsock/pkg/transport"
+)
+
+func checkIfCRCDaemonRunning() error {
+	l, err := transport.Listen(transport.DefaultURL)
```

Comment:
It seems a little bit brutal. Perhaps try to connect to the daemon with the crc.sock (the binary protocol) or to transport.DefaultURL with http.